common/app/views/fragments/immersiveMainMedia.scala.html

```diff
@@ -87,9 +87,18 @@
             <div class="gs-container">
                 @if(!isTheMinuteArticle) {
                     <div class="content__main-column">
-                    @fragments.meta.metaInline(page.item)
+                    @if(!isPaidContent){
```

Comment:
in [there](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/meta/metaInline.scala.html#L24), no?